### ISSUE TITLE
Uploading large images with `percy upload`

### ISSUE THREAD

**Issue by danielschleindlsperger** [percy/percy-agent#459]
Hey!

I just stumbled upon the problem where I want to upload a large image with `percy upload` and I'm getting the error
```sh
[percy] StatusCodeError 400 - {"errors":[{"status":"bad_request","detail":"The minimum-height attribute must be between 10 and 2000 inclusive. Received 6465."}]} | Fri Jan 31 2020 18:26:48 GMT+0100 (Central European Standard Time)
```

I think percy-agent sets the minimum-height based on the height of the image, which then fails backend validation.

I can't upload an HTML snapshot, since some of the assets are on my company's private staging system behind a firewall. So I figured I would just manually take the screenshots and upload them to percy for the diffing and approval workflow goodies.

Maybe I'm doing something wrong or there is another workaround I'm not thinking of?

Thanks in advance and have a nice weekend!

**Comment by Robdel12** [comment]
Hey Daniel,

Thanks for the report! I'm going to side step the `upload` stuff for a second and see if we can address this first (and then maybe we don't have to worry about `upload`). 

> I can't upload an HTML snapshot, since some of the assets are on my company's private staging system behind a firewall. So I figured I would just manually take the screenshots and upload them to Percy for the diffing and approval workflow goodies.

We support [capturing many asset hosts](https://docs.percy.io/docs/capturing-multiple-asset-hosts) in our SDKs, I'd be curious if that would work for you here. It sounds like the exact use case we built it for. 😃 What that will do is save those assets from the hosts you specify so they're locally available in our rendering env.

**Comment by danielschleindlsperger** [comment]
Thanks for the speedy response! 

Capturing asset hosts would actually work perfectly if our site were fully responsive 😅 
We're already using Percy with Storybook to responsively test components (although we had to use another workaround here since I don't belive `percy-storybook` uses the sdk configuration).
Right now I'm trying to snapshot full pages for mobile, desktop and AMP (each with specific viewport width, user-agent and url)which doesn't quite fit with the responsive viewport `widths` model of Percy.

Anyway it seems Wil has already fixed it in the CLI 😁 🙏

**Comment by wwilsman** [comment]
PR was just merged. 😃 

Should be released now as v0.20.13

**Comment by danielschleindlsperger** [comment]
Perfect, thank you so much! 🙏
